### DISCUSSION TITLE
Setting buster at wagmi.cache

### DISCUSSION THREAD

**Discussion by oskarvu** [wevm/wagmi#2692] [Q&A]
Hey! Is there a way to set a `buster` prop at `wagmi.cache`?

For a context: I recently updated wagmi from 0.12 to the latest, and app started throwing some exceptions. After clearing local storage, issues disappeared.

**Comment by Yuripetusko** [comment]
Updated to latest wagmi (1.x.x) and getting errors too, deleting `wagmi.cache` solves it 🤔  would be nice to set the buster/version

**Comment by Yuripetusko** [comment]
Basically react-query cached responses have BigNumbers still, so when upgrading to new version which uses native bigints, some of the code that now expects BigInt will not work, and this is one of the examples. So we need some sensible way to change cache version or cache busting or clear localstorage `wagmi.cache` when releasing dapps with updated wagmi
